doc/Extensions/Applications.md

```diff
@@ -168,6 +169,17 @@ extend memcached /usr/local/bin/memcached
 4. Restart snmpd on your host
 5. On the device page in Librenms, edit your host and check `Memcached` under the Applications tab.
 
+### Proxmox
+1. Copy the [proxmox script](https://github.com/librenms/librenms-agent/blob/master/agent-local/proxmox) to `/usr/local/bin` (or any other suitable location) on your remote server.
```

Comment:
I'd add a curl command to download the raw file directly to /usr/local/bin/proxmox